backend/internal/model/init.go

```diff
@@ -25,15 +25,17 @@ type Competition struct {
 	HostID 			uint `gorm:"not null"`
 	ScoreboardURL	string
 	Overview		string
-	MenRecurve 		bool
-	WomenRecurve 	bool
-	MenCompound 	bool
-	WomenCompound 	bool
+}
+
+type CompetitionCategory struct {
+	CompetitionID 	uint `gorm:"not null"`
+	Description 	string  `gorm:"not null"`
+	Distance 		int32 `gorm:"not null"`
 }
 
 type Participant struct {
-	UserID uint `gorm:"primaryKey"`
-	CompetitionID uint `gorm:"not null"`
+	UserID 			uint `gorm:"not null"`
```

Comment:
Why not primary key?

Reply:
A user can join in multiple competitions, and it seems that GORM assumes the primaryKey is unique(?)

Reply:
Make sense